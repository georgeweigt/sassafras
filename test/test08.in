* Salinity data from STAT 650 ;

data ;
input y x ;
x2=x**2 ;
x3=x**3 ;
datalines ;
59.5 1.6
53.3 1.6
56.8 1.6
63.1 1.6
58.7 1.6
55.2 3.8
59.1 3.8
52.8 3.8
54.5 3.8
51.7 6.0
48.8 6.0
53.9 6.0
49.0 6.0
44.6 10.2
48.5 10.2
41.0 10.2
47.3 10.2
46.1 10.2
;

proc reg ;
model y = x ;

proc reg ;
model y = x x2 x3 ;

