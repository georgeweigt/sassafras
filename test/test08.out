                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      1          453.04692       453.04692       56.65     0.0000
     Error     16          127.95586         7.99724                       
     Total     17          581.00278                                       

              Root MSE            2.82794     R-Square     0.7798
              Dependent Mean     52.43889     Adj R-Sq     0.7660
              Coeff Var           5.39283                        

                              Parameter Estimates

                        Estimate     Std Err     t Value     Pr > |t|
          Intercept     60.66998     1.28072       47.37       0.0000
          X             -1.50875     0.20045       -7.53       0.0000

                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      3          456.50478       152.16826       17.11     0.0001
     Error     14          124.49800         8.89271                       
     Total     17          581.00278                                       

              Root MSE            2.98207     R-Square     0.7857
              Dependent Mean     52.43889     Adj R-Sq     0.7398
              Coeff Var           5.68674                        

                              Parameter Estimates

                        Estimate     Std Err     t Value     Pr > |t|
          Intercept     58.06731     6.42753        9.03       0.0000
          X              0.94982     5.15107        0.18       0.8563
          X2            -0.56574     1.08093       -0.52       0.6089
          X3             0.03449     0.06286        0.55       0.5919

